types/slice.go

```diff
@@ -0,0 +1,172 @@
+package types
+
+import (
+	"strconv"
+	"strings"
+
+	"github.com/drone/ff-golang-server-sdk/log"
+)
+
+// Slice type for clause attribute evaluation
+type Slice struct {
+	data interface{}
+}
+
+// NewSlice creates a Slice instance with the object value
+func NewSlice(value interface{}) Slice {
+	return Slice{
+		data: value,
+	}
+}
+
+// In compares the attributes held by the slice with the input values.
+// it will first determine what type of slice this is, before casting values
+// to the appropriate type.
+func (s Slice) In(values []string) bool {
+	// Determine what kind of slice we have
+	switch s.data.(type) {
```

Comment:
data is interface{} and that can be a pointer, what do you think do we need to check for pointer on slice?

Reply:
Yes - I'll add a test case for this as well.